src/globalPrototype.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable no-extend-native */
+Array.prototype.forEachAsync = async function (fn) {
+    for (let t of this) { await fn(t) }
+};
+
+Array.prototype.forEachAsyncParallel = async function (fn) {
+    await Promise.all(this.map(fn));
```

Comment:
Array.foeEach looks better then map in this context, if I got the point :)

Reply:
Oh, we use it in Promise.all, nevermond. I would suggest to move this function out and use as helper in case you need it.